oss-internship-2020/curl/README.md

```diff
@@ -0,0 +1,63 @@
+# Curl Sandboxed
+
+This library is a sandboxed version of curl's C API, [libcurl](https://curl.haxx.se/libcurl/c/), implemented using Sandboxed API.
+
```

Comment:
Please add a section detailing how to build this. You should also mention `git submodule update --init --recursive` for the submodule :)